modules/apps/calendar/calendar-web/src/main/resources/META-INF/resources/js/legacy/scheduler.js

```diff
@@ -408,18 +408,14 @@ AUI.add(
 						titleCurrentValue: '',
 					};
 
-					Liferay.Util.openWindow({
-						dialog: {
-							after: {
-								destroy() {
-									instance.load();
-								},
-							},
-							destroyOnHide: true,
-							modal: true,
+					Liferay.Util.openModal({
```

Comment:
Does this mean that we will be able to remove `openWindow` some day because it's superseded by `openModal`? :thinking:

Reply:
Yeah, we should've replaced it long ago..